packages/bits/src/lib/selector/selector.component.ts

```diff
@@ -105,6 +105,7 @@ export class SelectorComponent implements OnChanges, AfterViewInit, OnDestroy, I
         this.overlay.clickOutside
             .pipe(takeUntil(this.onDestroy$))
             .subscribe(_ => this.overlay.hide());
+        this.checkbox.checkboxLabel.nativeElement.setAttribute("tabindex", "-1");
```

Comment:
Ideally, there would be an input on the checkbox component to do this behavior. Do you mind creating a ticket to make this improvement in the future?